src/operator/nn/pooling.cc

```diff
@@ -92,6 +92,9 @@ static bool PoolingShape(const nnvm::NodeAttrs &attrs,
                          std::vector<TShape> *out_shape) {
   const PoolingParam &param = nnvm::get<PoolingParam>(attrs.parsed);
   CHECK_EQ(in_shape->size(), 1U);
+  if (param.pool_type == pool_enum::kLpPooling) {
+    CHECK(param.p_value.has_value());
```

Comment:
We'd better not do this as this may affect Jun's work. If we set p_value to be defaulting to 1 instead of optional, the generated json file for this symbol would contain an extra field.